backend/src/packages/auth/auth.service.ts

```diff
@@ -66,6 +82,53 @@ class AuthService {
 
     return { user, token };
   }
+
+  public async sendEmailResetPasswordLink(
+    authRequestPasswordDto: AuthRequestPasswordDto,
+    url: string,
+  ): Promise<SendEmailResponse> {
+    const { email } = authRequestPasswordDto;
+    const user = await this.userService.findByEmail(email);
+    if (!user) {
+      throw new UserNotFoundError();
+    }
+    const token = await accessToken.create<{ userId: number }>(
+      {
+        userId: user.id,
+      },
+      '1h',
```

Comment:
Let's create TokenExpirationTime enum and store values needed for token service there
Like
```
const TokenExpirationTime = {
  ONE_DAY = '24h',
  ONE_HOUR = '1h'
}
```

Reply:
I did it but looks like I can't use it in config because of circular dependencies

Reply:
So should we move this enum outside of token package or leave "24h" in config ?